mmv1/third_party/terraform/services/vmwareengine/resource_vmwareengine_cluster_test.go

```diff
@@ -70,27 +70,7 @@ func TestAccVmwareengineCluster_vmwareEngineClusterUpdate(t *testing.T) {
 func testVmwareEngineClusterConfig(context map[string]interface{}, nodeCount int) string {
 	context["node_count"] = nodeCount
 	return acctest.Nprintf(`
-resource "google_project" "project" {
-  project_id      = "tf-test%{random_suffix}"
-  name            = "tf-test%{random_suffix}"
-  org_id          = "%{org_id}"
-  billing_account = "%{billing_account}"
-}
-
-resource "google_project_service" "vmwareengine" {
-  project = google_project.project.project_id
-  service = "vmwareengine.googleapis.com"
-}
-
-resource "time_sleep" "sleep" {
```

Comment:
`TestAccVmwareengineCluster_vmwareEngineClusterUpdate` failed with
```
 vcr_utils.go:152: Step 1/6 error: Error running pre-apply refresh: exit status 1
        Error: Reference to undeclared resource
          on terraform_plugin_test.tf line 10, in resource "google_vmwareengine_network" "cluster-nw":
          10:     time_sleep.sleep # Sleep allows permissions in the new project to propagate
        A managed resource "time_sleep" "sleep" has not been declared in the root
        module.
```

Looks like we removed the time_sleep resource since it's not needed any more as we aren't creating new projects to wrap the test. You'll need to remove the dependency on time_sleep from the google_vmwareengine_network config as well